crates/api/proto/op_store.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package kitsune2.op_store;
+
+// An op.
+message Op {
+    // Op id.
+    bytes op_id = 1;
```

Comment:
Want to question the value of sending the `op_id` with the `Op`. When we're communicating a diff, we have to work in terms of `op_id`s but when we have the `op_data`, we shouldn't trust the `op_id` that's been provided, we should calculate it from the `op_data`. This links to what we were discussing on the DHT diff PR where hash length mismatches would cause an error, we need to be properly checking these as they arrive.

Reply:
That's true. I'll refactor.

Reply:
Wait, we don't have a hashing for that yet tho. Should I add some temporary hashing for the id?

Reply:
Yes please. I had to do something similar with receiving ops into the store vs notifying the DHT model that ops have arrived on the host. Maybe it needs to be a proper test utility as part of demonstrating the host. I can't remember exactly where I did this because it's been a couple of weeks, but I think I skipped actually calculating hashes.

Reply:
I'll have to compute hashes if the ops that are requested by id and then sent without id need to be compared. I can add that as a test utility.

Reply:
I added it for this test only so far. The hash function will sooner or later be added globally. We can refactor this then to use it.